src/test/java/hudson/plugins/cobertura/CoberturaPublisherTest.java

```diff
@@ -1,49 +0,0 @@
-package hudson.plugins.cobertura;
```

Comment:
Is there a reason why this test was removed? I assume it was because of the change in the constructor?